pkgs/by-name/li/libgff/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
You should use the new [`finalAttrs`](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes) idiom.